hazelcast/src/test/java/com/hazelcast/aggregation/MapAggregatePerformanceTest.java

```diff
@@ -0,0 +1,176 @@
+package com.hazelcast.aggregation;
+
+import com.hazelcast.aggregation.impl.DoubleAverageAggregator;
+import com.hazelcast.config.Config;
+import com.hazelcast.config.InMemoryFormat;
+import com.hazelcast.config.MapConfig;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.IMap;
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.nio.serialization.DataSerializable;
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.TestHazelcastInstanceFactory;
+import com.hazelcast.test.annotation.ParallelTest;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({QuickTest.class, ParallelTest.class})
+public class MapAggregatePerformanceTest extends HazelcastTestSupport {
+
+    @Test
+    @Ignore("needs 10G of heap to run")
```

Comment:
the `Ignore` will still run it, right? except it won't fail the build when the test fails?